dubbo-admin-backend/src/main/java/org/apache/dubbo/admin/common/AdminConstants.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.dubbo.admin.common;
+
+/**
+ * @author zhanghuilong
+ * @desc   Constants
+ * @since 2019/01/03
+ */
+public class AdminConstants {
+
+    public static final String PUNCTUATION_POINT = ".";
+
+    public static final String PUNCTUATION_SEPARATOR_POINT = "\\.";
+
+    public final static String PATH_SEPARATOR = "/";
+
+    public final static String INTERROGATION_POINT = "?";
+
+    public static final String ANY_VALUE = "*";
+
+    public static final String APPLICATION = "application";
+
+    public static final String SERVICE = "service";
+
+    public static final String IP = "ip";
+}
```

Comment:
We already have a class: `org.apache.dubbo.admin.common.util.Constants`,  please add these variables there.

Reply:
ok , thank you. Later I will add these again.